auslib/db.py

```diff
@@ -1802,10 +1828,13 @@ def countReleases(self, transaction=None):
         return count
 
     def getReleaseInfo(self, name=None, product=None, limit=None,
-                       transaction=None, nameOnly=False, name_prefix=None):
+                       transaction=None, nameOnly=False, name_prefix=None, batch=False):
```

Comment:
This would be cleaner/faster if you changed `name` to `names` and used "WHERE IN" instead of a union. You'll need to update a few callers to cope with the change, but I think it's worth it.

Reply:
Yeah, great Idea. Will do this ASAP. 👍